io.github.randovania.Randovania.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
This file should be maintained in your repo and be included in the tarball.

Reply:
What is the expected path inside the tarball?

Reply:
I guess it doesn't matter, as long as it ends up installed in the right place.

Reply:
Would be it enough to have a .desktop file that's just like this, but with the Exec line set to...
```
Exec=%REPLACE_ME_WITH_YOUR_PATH_PLEASE%
```
... and then replace it in the `build-commands`?

Reply:
Doesn't even need to be that, you can use desktop-file-edit to set Exec to whatever is needed.